rest-api-spec/src/yamlRestTest/resources/rest-api-spec/test/logsdb/10_settings.yml

```diff
@@ -68,6 +68,72 @@ create logs index:
   - is_true: test
   - match: { test.settings.index.mode: "logsdb" }
 
+---
+create logs index without doc values sparse index:
```

Comment:
Maybe also add a test that doesn't set `index.mapping.use_doc_values_skipper` index setting and check that `@timestamp` does't have points.